.github/workflows/triton.yml

```diff
@@ -0,0 +1,99 @@
+on:
```

Comment:
The problem with using the existing GPU flow is that it build pytorch without cuda dependency which doesn't work with Triton as Triton uses PyTorch to detect if there is a GPU device.

It would be great if we can use this for now while I figure out how to merge the two CI workflows.

Reply:
Same comment as the triton.sh. @will-cromar

Reply:
Is it feasible to  merge your new tests into the existing workflow and upgrade the GPUs we use there? It's probably time we use more modern GPUs for all of our CI tests anyway.

Reply:
You can upgrade the runner here: https://github.com/pytorch/xla/blob/8a1ada88e4f657c83d660b32e775e564fb8ea320/.github/workflows/build_and_test.yml#L56

Reply:
Let me check with pytorch folks first..